lib/warder/version.rb

```diff
@@ -1,4 +1,4 @@
 # define warder version
 module Warder
-  VERSION = '0.2.0'
+  VERSION = '0.2.1'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.